### ISSUE TITLE
no such file or directory: _routes.json 

### ISSUE THREAD

**Issue by adamdehaven** [harlan-zw/nuxt-seo#333]
### Describe the bug

When upgrading `@nuxtjs/seo` from `2.0.0-rc.21` to `2.0.0-rc.23` I get the following error when running `build`:

```shellsession
no such file or directory, open '/apps/{name}/.output/public/_routes.json'  
```

I believe it originated from one of the commits in `2.0.0-rc.22` listed here:
https://github.com/harlan-zw/nuxt-seo/releases/tag/v2.0.0-rc.22

I've isolated the change to the `2.0.0-rc.22` release as it's the only package I'm bumping in (via Renovate) with pnpm when the build failure occurs.

If I downgrade back to `2.0.0-rc.21` the error disappears.

### System / Nuxt Info

- Operating System: `Darwin`
- Node Version:     `v20.17.0`
- Nuxt Version:     `3.13.1`
- CLI Version:      `3.13.2`
- Nitro Version:    `2.9.7`
- Package Manager:  `pnpm@9.11.0`
- Builder:          `-`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

**Comment by harlan-zw** [comment]
This is from the latest OG Image that is optimizing the Cloudflare rules. Seems like the routes.json file isn't being generated from Nitro for some reason, I've pushed a fix in 3.0.6 to allow it not to exist, however, it may be a nitro preset that I haven't properly handled.

Are you able to let me know your nitro preset exactly?

**Comment by adamdehaven** [comment]
```ts
nitro: {
  preset: 'cloudflare-module',
  prerender: {
    // Set to false to match Cloudflare route matching rules: https://nuxt.com/deploy/cloudflare
    autoSubfolderIndex: false,
  },
},
```
